src/components/Cards/CollectionCard.tsx

```diff
@@ -0,0 +1,118 @@
+import { Badge, Group, Stack, Text } from '@mantine/core';
+import { IconLayoutGrid } from '@tabler/icons-react';
+import { useCardStyles } from '~/components/Cards/Cards.styles';
+import { FeedCard } from '~/components/Cards/FeedCard';
+import { EdgeImage } from '~/components/EdgeImage/EdgeImage';
+import { IconBadge } from '~/components/IconBadge/IconBadge';
+import { ImageGuard } from '~/components/ImageGuard/ImageGuard';
+import { MediaHash } from '~/components/ImageHash/ImageHash';
+import { DEFAULT_EDGE_IMAGE_WIDTH } from '~/server/common/constants';
+import { CollectionGetInfinite } from '~/types/router';
+import { abbreviateNumber } from '~/utils/number-helpers';
+import { isDefined } from '~/utils/type-guards';
+
+export function CollectionCard({ data }: Props) {
+  const { classes, cx } = useCardStyles({ aspectRatio: 1 });
+
+  const getCoverImages = () => {
+    if (data.image) return null;
+
+    return data.items
+      .map((item) => {
+        switch (item.type) {
+          case 'model':
+          case 'post':
+            return item.data.image;
+          case 'image':
+            return item.data;
+          case 'article':
+          default:
+            return null;
+        }
+      })
+      .filter(isDefined);
+  };
+
+  const coverImages = getCoverImages();
+
+  return (
+    <FeedCard
+      className={classes.noImage}
+      href={`/collections/${data.id}`}
+      aspectRatio="square"
+      sx={{ position: 'absolute', top: 0, right: 0, bottom: 0, left: 0 }}
+    >
+      <div className={classes.root}>
```

Comment:
Might wanna use the noImage class like in the Model card when Image is null

Reply:
It's been used up above in the `FeedCard`